extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -45,49 +48,63 @@ oppia.directive('oppiaInteractiveGraphInput', [
             isWeighted: false,
             isLabeled: false
           };
-
           $scope.submitGraph = function() {
             // Here, angular.copy is needed to strip $$hashkey from the graph.
             $scope.onSubmit({
               answer: angular.copy($scope.graph),
               rulesService: graphInputRulesService
             });
           };
+          $scope.interactionIsActive = ($scope.getLastAnswer() === null);
+          $scope.$on(EVENT_NEW_CARD_AVAILABLE, function(evt, data) {
+            $scope.interactionIsActive = false;
```

Comment:
Consider adding a newline below this one, since this line is kind of separate from the rest.

Reply:
Done